src/vic-stdlib.c

```diff
@@ -0,0 +1,45 @@
+#ifndef _VIC_STDLIB
+#define _VIC_STDLIB
+
+#include "vic.h"
+#include "stdlib.h"
+
+void vic_set(void)
+{
+    char var_name[VIC_VAR_NAME_LEN+1] = {'\0'};
+    char var_val[VIC_VAR_VAL_LEN+1] = {'\0'};
+    char* args_format = \
+        "%"VIC_XSTR(VIC_VAR_NAME_LEN)"s %"VIC_XSTR(VIC_VAR_VAL_LEN)"s";
```

Comment:
@vargac this looks pretty nasty but I do not really see a nicer way of doing this right away.